src/controllers/index.js

```diff
@@ -0,0 +1,9 @@
+import joinRandomGame from './join-random-game';
```

Comment:
Also, to add a new message this now requires you to edit two files, one for the actual file containing the handler and adding the import statement here where as before you only had to add the file.